tests/resources/atomic.json

```diff
@@ -1,92 +1,85 @@
-                    "type": "INT64"
-                },
-                "json": {
-                    "type": "integer"
-                }
-            }
+  "name": "atomic",
+  "tests": [
+    {
+      "name": "test_atomic",
+      "test": {
+        "avro": {
+          "type": "long"
         },
-        {
-            "name": "test_atomic_with_null",
-            "test": {
-                "avro": [
-                    {
-                        "type": "null"
-                    },
-                    {
-                        "type": "long"
-                    }
-                ],
-                "bigquery": {
-                    "mode": "NULLABLE",
-                    "type": "INT64"
-                },
-                "json": {
-                    "type": [
-                        "integer",
-                        "null"
-                    ]
-                }
-            }
+        "bigquery": {
+          "mode": "REQUIRED",
+          "type": "INT64"
         },
-        {
-            "description": [
-                "Test that overlapping types are treated as json blobs."
-            ],
-            "name": "test_incompatible_atomic_multitype",
-            "test": {
-                "avro": {
-                    "type": "string"
-                },
-                "bigquery": {
-                    "mode": "REQUIRED",
-                    "type": "STRING"
-                },
-                "json": {
-                    "type": [
-                        "boolean",
-                        "integer"
-                    ]
-                }
-            }
+        "json": {
+          "type": "integer"
+        }
+      }
+    },
+    {
+      "name": "test_atomic_with_null",
+      "test": {
+        "avro": [
+          {
+            "type": "null"
+          },
+          {
+            "type": "long"
+          }
+        ],
+        "bigquery": {
+          "mode": "NULLABLE",
+          "type": "INT64"
+        },
+        "json": {
+          "type": [
+            "integer",
+            "null"
+          ]
+        }
+      }
+    },
+    {
+      "name": "test_incompatible_atomic_multitype",
+      "test": {
+        "avro": {
+          "type": "string"
+        },
+        "bigquery": {
+          "mode": "REQUIRED",
+          "type": "STRING"
+        },
+        "json": {
+          "type": [
+            "boolean",
+            "integer"
+          ]
+        }
+      }
+    },
+    {
+      "name": "test_incompatible_atomic_multitype_with_null",
+      "test": {
+        "avro": [
+          {
+            "type": "null"
+          },
+          {
+            "type": "string"
+          }
+        ],
+        "bigquery": {
+          "mode": "NULLABLE",
+          "type": "STRING"
         },
-        {
-            "description": [
-                "Test that overlapping types that can be null are nullable json blobs.",
-                "A field is null if any of it's types are null."
```

Comment:
This field is dropped during the serde process (`TestCase::description`)

Reply:
😢

Reply:
added serializer options to preserve description